src/command-parser/expand-arguments.ts

```diff
@@ -10,6 +10,8 @@ export class ExpandArguments implements CommandParser {
     constructor(private readonly additionalArguments: string[]) {}
 
     parse(commandInfo: CommandInfo) {
+        const configuration = getShellConfigurationSync();
```

Comment:
Looked at the source of this function briefly, seems it defaults to the value of `COMSPEC`.
Since cmd.exe is hardcoded for windows (source below), should the same be done here?

https://github.com/open-cli-tools/concurrently/blob/e52d984fa179e90b2d8f3548012e38987500a8a3/src/spawn.ts#L17-L18